GetRawYieldsDplusDs.py

```diff
@@ -50,6 +52,10 @@
 ptMaxs = fitConfig[cent]['PtMax']
 fixSigma = fitConfig[cent]['FixSigma']
 fixMean = fitConfig[cent]['FixMean']
+if 'EnableRef' not in fitConfig[cent]:
+    enableRef = False
+else:
+    enableRef = fitConfig[cent]['EnableRef']
```

Comment:
This conflict with the C++ macro, which crashes if no `EnableRef` entry is present in the yml file. 
For convention, we add all the options in the yml files. Please update all the configuration files for the fit for all the particles adding the new parameter.

Reply:
Ok